src/index.js

```diff
@@ -39,18 +39,23 @@ static addTask() {
   if (recive !== '') {
     const newTask = {
       description: recive,
-      completed: false,
-      index: 0,
+      completed: true,
```

Comment:
I think I had well implemented this completed from the first time I think the first code reviewer confused me, I had it set to false, if you look at what I suggested, because for this step of add and remove is not necessary to be functional that is for the final task which I have not started, as I have already corrected it, please ask you to review it again thank you, so you do not have to make a new request review